pkg/kubectl/cmd/alpha.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"io"
+
+	"github.com/spf13/cobra"
+
+	"k8s.io/kubernetes/pkg/kubectl/cmd/templates"
+	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+	"k8s.io/kubernetes/pkg/util/i18n"
+)
+
+// NewCmdAlpha creates a command that acts as an alternate root command for features in alpha
+func NewCmdAlpha(f cmdutil.Factory, in io.Reader, out, err io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "alpha",
```

Comment:
I'd be happy to do that, but alpha is a little bit different from set/rollout/config as its more of a container for commands rather than a command with subcommands. For example, `config get` and `config set` are related subcommands that operate on `config`, but `alpha debug` and `alpha foo` are unrelated other than both being in an alpha state.

Also, the commands in alpha could themselves have subcommands, which would make the "Usage: kubectl alpha SUBCOMMAND [options]" not quite correct. 

As its coded now, without a run function, Usage will be omitted from help and look like this:

```
These commands correspond to alpha features that are not enabled in Kubernetes clusters by default.                                                                            
                                                                                                                                                                               
Available Commands:                                                                                                                                                            
  debug       Run a debug container in a pod                                                                                                                                   
                                                                                                                                                                               
Use "kubectl <command> --help" for more information about a given command.   
```

If we were to give it a default run function and set Use to  "alpha COMMAND", then the help would be:
```
These commands correspond to alpha features that are not enabled in Kubernetes clusters by default.                                                                            
                                                                                                                                                                               
Available Commands:                                                                                                                                                            
  debug       Run a debug container in a pod                                                                                                                                   
                                                                                                                                                                               
Usage:                                                                                                                                                                         
  kubectl alpha COMMAND [options]                                                                                                                                           
                                                                                                                                                                               
Use "kubectl <command> --help" for more information about a given command.                                                                                                     
Use "kubectl options" for a list of global command-line options (applies to all commands). 
```